src/functional/otf/binding/pybind.py

```diff
@@ -34,13 +33,17 @@ class DimensionType(Expr):
     name: str
 
 
-class SidConversion(Expr):
-    buffer_name: str
+class BufferIterator(Expr):
```

Comment:
I'd remove `Iterator` here, because they are not iterators (in the iterator ir sense). The iterators used in the backend are wrapped around these objects.

Reply:
Renamed it to `BufferSID` and `PositionalSID`.